.github/workflows/build-and-test.yaml

```diff
@@ -507,20 +506,15 @@ jobs:
       - package
       - params
     outputs:
-      oci_image_ref: ${{ steps.prepare.outputs.oci_image_ref }}
-      ocm_resources: ${{ steps.prepare.outputs.ocm_resources }}
+      oci_image_ref: ${{ steps.image-build.outputs.image_reference }}
+      ocm_resources: ${{ steps.image-build.outputs.ocm_resource }}
```

Comment:
nit: Let us rather consistently use singular here, i.e. `ocm_resource`.

Reply:
@8R0WNI3 sure, we should eventually refactor this part, as well.

rationale behind using plural forms consistently within this workflow: there are multiple jobs each emitting either one ore multiple ocm-resources + the `ocm append resource` command is built to allow both.

hence, using plural form made sense to me for jobs defined in workflow (as opposed to the (new) action emitting only one resource / image)

at any rate, it will still have to be seen how we can best support handling of multiple resources, considering [limitations of matrix-build](https://github.com/orgs/community/discussions/17245), for which there are already some workarounds like [this](https://github.com/cloudposse/github-action-matrix-outputs-write).